### PR TITLE
[event] add sprint page late november

### PR DESCRIPTION
**Pull request by tomatoprinx**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (yungyuc) [inline .vscode/settings.json:1]: Why do you include this file?

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/sprint/index.rst [hunk 2]
```diff
@@ -12,6 +12,8 @@ hosted by sciwork.
 
 Scisprint 2021
 ==============
+* `Scisprint 2021 Late November: Python ZH TW Doc
```
Comment (yungyuc) [inline content/pages/sprint/index.rst:15]: There should be a blank line between the list and the header.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/sprint/2021/11late-pycht.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
+=================================================
+Scisprint 2021 Late November: Python ZH TW Doc
```
Comment (yungyuc) [inline content/pages/sprint/2021/11late-pycht.rst:2]: Can you make it explicitly that this is online?  There is an in-person sprint being planned on a close date.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/sprint/2021/11late-pycht.rst (hunk 3 above)
Comment (tomatoprinx) [reply content/pages/sprint/2021/11late-pycht.rst:2]: @yungyuc you mean like this title?

```markdown
=================================================
(Online) Scisprint 2021 Late November: Python ZH TW Doc
=================================================

...
```

Comment (yungyuc) [reply content/pages/sprint/2021/11late-pycht.rst:2]: > @yungyuc you mean like this title?
> 
> 
> 
> ```markdown
> 
> =================================================
> 
> (Online) Scisprint 2021 Late November: Python ZH TW Doc
> 
> =================================================
> 
> 
> 
> ...
> 
> ```

Yes.

BTW there will be an in-person sprint hosted on the Saturday: https://discord.com/channels/730297880140578906/730393342818648204/905423996550529064

I will be prepping for a PR for it.  FYI.

Comment (yungyuc) [reply content/pages/sprint/2021/11late-pycht.rst:2]: See #88

Comment (tomatoprinx) [reply content/pages/sprint/2021/11late-pycht.rst]: Okay I'll merged #88  to this one

**Pull request merged**